hyperspy/models/eelsmodel.py

```diff
@@ -538,31 +538,92 @@ def _fit_edge(self, edgenumber, start_energy=None, **kwargs):
         # Recover the channel_switches. Remove it or make it smarter.
         self.channel_switches = backup_channel_switches
 
-    def quantify(self):
-        """Prints the value of the intensity of all the independent
-        active EELS core loss edges defined in the model
+    def quantify(self, output="absolute"):
+        """Prints and returns the values of the intensity of all the independent
+        active EELS core loss edges defined in the model.
 
+        Parameters
+        ----------
+        output : 'absolute', 'normalized', or None
+            If 'absolute', prints values of the intensity parameter of all EELSCLEdge components.\n
+            If 'normalized', prints intensity parameters normalized with respect to the sum of all intensities in the model. If all relevant elements are present in the analyzed spectrum this results in the elemental concentrations in atomic %.\n
+            If None, no output is printed.
+
+        Returns
+        -------
+        dict
+            Python dictionary with element name, edge name and intensity value.
+
+        Examples
+        -------
+        Load and fit the BN EELS example from the user guide:\n
+        >>> s = hs.datasets.eelsdb(title="Hexagonal Boron Nitride", spectrum_type="coreloss")[0]
+        >>> ll = hs.datasets.eelsdb(title="Hexagonal Boron Nitride", spectrum_type="lowloss")[0]
+        >>> s.add_elements(('B', 'N'))
+        >>> m = s.create_model(ll=ll)
+        >>> m.enable_fine_structure()
+        >>> m.smart_fit()
+
+        Print the elemental concentrations:\n
+        >>> m.quantify(output='normalized')
+
+        Calculate B/N ratio with the returned dictionary:\n
+        >>> q = m.quantify(output=None)
+        >>> print('B/N ratio: ', q['B']['K']/q['N']['K'])
+
+        The intensity values can also be accessed directly from the model component:\n
+        >>> print('Intensity of B K edge: ', q['B']['K'])
+        >>> print('Intensity of B K edge: ', m.components.B_K.intensity.value)
+
+        See also
+        ---------
+        :py:class:`~._components.eels_cl_edge.EELSCLEdge`
         """
         elements = {}
+        if output == "normalized":
+            int_sum = 0
         for edge in self._active_edges:
             if edge.active and edge.intensity.twin is None:
                 element = edge.element
                 subshell = edge.subshell
                 if element not in elements:
                     elements[element] = {}
                 elements[element][subshell] = edge.intensity.value
+                if output == "normalized":
+                    int_sum += edge.intensity.value
+
+        if output == None:
+            return elements
         print()
-        print("Absolute quantification:")
-        print("Elem.\tIntensity")
+        if output == "absolute":
+            print("Absolute quantification:")
```

Comment:
A suggestion for improvement: use prettytable to get a nicer display? For example:
https://github.com/hyperspy/hyperspy/blob/bb684ee33292e99e2bae2eb9ecd39f8506360164/hyperspy/_signals/eels.py#L242-L266

and 

https://github.com/hyperspy/hyperspy/blob/7ca78caac1fcae55d4be672f51632ba20504d418/hyperspy/utils/__init__.py#L71-L91

Reply:
Sounds good! I will look into that.